### DISCUSSION TITLE
Will I really lose my reward?

### DISCUSSION THREAD

**Discussion by qianmyth** [Chia-Network/chia-blockchain#6297] [Support]
I often receive a lot of such information? Will I lose my reward? How to improve?

2021-05-31T13:05:08.564 harvester chia.harvester.harvester: WARNING  Looking up qualities on /dest/group-100/plot-k32-2021-05-21-11-05-ba3e9ff7c411763062c252026b6bcc2de3bec75ab9a38789f84eade4f23e61e3.plot took: 5.105397939682007. This should be below 5 seconds to minimize risk of losing rewards.

**Comment by room101-dev** [comment]
> I often receive a lot of such information? Will I lose my reward? How to improve?
> 
> 2021-05-31T13:05:08.564 harvester chia.harvester.harvester: WARNING Looking up qualities on /dest/group-100/plot-k32-2021-05-21-11-05-ba3e9ff7c411763062c252026b6bcc2de3bec75ab9a38789f84eade4f23e61e3.plot took: 5.105397939682007. This should be below 5 seconds to minimize risk of losing rewards.

Yes, this was put into CHIA by the dev-team ( aka Bangalore subcontractors )  about 2 weeks ago, HPOOL sent out a warning telling everybody 'its not their fault' that CHIA-NET dev is insane, and intends to kill all the players, except the insiders who can stay up with the game. Only rich can stay in the game. Golden Rule, he with the Gold makes the Rules.

The deal is this, with the new 5 second rule, you can no longer farm with slow drives, or usb, or on a lan ( sas, anything network is too slow )

This is a major game changer, it means only those with the newest HDD's and the fastest get to play the game.

So much for used hw, old hw, ... everything that they ever said was always a LIE.

**Reply by qianmyth** [reply]
If the harvester accesses the file for more than 5 seconds, there will be a risk. How many seconds will it lose the reward?

**Comment by room101-dev** [comment]
How to improve? Only 'farm' from fast disks that are physically attached to the machine, say 7200 rpm drives that are connected to the sata ports or pcie connector boards with sas/sata ports, 

USB 2.0, or lan hdd's will no longer work under this new 'bar' to drop out all but the elite players.

Same as its always been have a mining rig control up to 1/2 dozen drives, 4 on the sata, and four on a pcie-sata extender card, then two nvme's on the mobo so that your cpu runs at full speed, note the 'chia plots check' is cpu&disk intensive to stay under the 5 second rule ( plot testing ) kills drives too

Yes, when chia say's "You Lose", its that rare time that they're not lying to you.

**Reply by XIII39** [reply]
Most HDDs can support under 1s lookup times. This error usually occurs when you are copying plots to the drive. The disk IO is high thus slowing lookup time. 

I am running multiple harvesters on a 10/100 network (old hardware), both SATA and SAS - 5400 and 7200 RPM drives and very seldom see lookup times greater than 1s or even 0.5s

Once you have copied all your plots and your drive is full, if you still see the error then there is another issue but, I'll bet there won't be.

**Reply by XIII39** [reply]
There error message should also be changed to read "This should be below 5 seconds to maximize chances of winning rewards"
The "This should be below 5 seconds to minimize risk of losing rewards" statement implies that you will lose rewards you have already won which is not the case.

**Reply by qianmyth** [reply]
If the harvester accesses the file for more than 5 seconds, there will be a risk. How many seconds will it lose the reward?

**Comment by joelrb** [comment]
It's recommended to separate your temp disk from your plot files. While plotting on the same disk where the plots are seek time is affected due to ongoing plotting read/write.

**Reply by qianmyth** [reply]
If the harvester accesses the file for more than 5 seconds, there will be a risk. How many seconds will it lose the reward?

**Comment by BrandtH22** [comment]
@qianmyth , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
